authentication.md

```diff
@@ -16,21 +16,21 @@ The framework integration will provide the application with some means of resolv
 
 * Reading the request headers for authentication information
 * Resolving the access token or API key presented to a Stormpath account
-* Return this information to the application, along with the method of authentication.
+* Return this information to the application, along with information about how the request was authenticated.
 
 This information will allow the application to make authorization decisions for the request. 
 
 **Unauthenticated Requests**
 
-In addition, the framework integration MAY provide the application with some means of rendering errors for unauthenticated requests. This may be combined with the Account Resolvers for extra convenience. 
+As a convenience, the framework integration MAY provide the application with a means to, in one step, check for authentication information and render an error if the request is unauthenticated. 
```

Comment:
Got it. It's not a problem; I'm simply pointing out that we made the earlier parts of the document much less implementation-specific, and this feels like adding some impl back in.

An alternative would be:

* Account Resolver: Is the request authenticated? If so, return who and how.
* Unauthenticated request behavior: Provide a way for the application to kick off the unauthenticated request logic.

It's not a blocking issue on this PR for me. It's the way it currently works in .NET, but that's not set in stone.